bootstrap/cabal.project

```diff
@@ -4,5 +4,4 @@
 packages: .
 optimization: False
 
-allow-newer:
```

Comment:
I think hardcoding the version here is not a good idea. TBH I am not entirely sure why it was needed.

Reply:
I removed this.